Neos.EventSourcedNeosAdjustments/Tests/Behavior/Features/EndToEnd/Changes/PropertyChange.feature

```diff
@@ -0,0 +1,32 @@
+Feature: Properties can be changed
+
+  Background:
+    Given I start with a clean database only once per feature
+    Given I execute the flow command "user:create" with the following arguments only once per feature:
+      | Name      | Value         |
+      | username  | admin         |
+      | password  | password      |
+      | firstName | A             |
+      | lastName  | D             |
+      | roles     | Administrator |
+    Given I execute the flow command "site:import" with the following arguments only once per feature:
+      | Name       | Value     |
+      | packageKey | Neos.Demo |
+    And I execute the flow command "contentrepositorymigrate:run" only once per feature
+    And I am logged in as "admin" "password"
```

Comment:
This line crashes (on the second invocation) with ` The given URI "http:///neos/login#" is not a valid one.` (which is strange I think)